tokio/src/runtime/basic_scheduler.rs

```diff
@@ -397,6 +402,10 @@ impl Spawner {
     pub(crate) fn reset_woken(&self) -> bool {
         self.shared.woken.swap(false, AcqRel)
     }
+
+    pub(crate) fn as_handle_inner(&self) -> &HandleInner {
+        &self.shared.handle_inner
+    }
```

Comment:
stupid naming nit, take it or leave it: i might have just called this `inner`?

Reply:
`inner()` on `BasicScheculer` would mean the basic scheduler's inner.

Reply:
oh, yeah, i missed that. carry on!